src/components/filesystem/index.js

```diff
@@ -426,6 +426,9 @@ module.exports = {
 
         goHome: function() {
             this.changePath("/");
+            Vue.nextTick(function() {
```

Comment:
Why isn't this being done the same as any other path change?

Reply:
On subsequent 'home' button clicks I can see that vue.js is creating a spinner after the directory is re-calculated. Put breakpoint on existing line console.debug('Spinner module created!'). I don't know why. Adding the Vue.nextTick line ensures that the spinner is hidden again.

Reply:
It doesn't make sense to me why this would only affect /.

Reply:
fixed a different way now

Reply:
Nice!